### ISSUE TITLE
Failing to render bold/italic greek symbols

### ISSUE THREAD

**Issue by MatiasLGonzalez** [KaTeX/KaTeX#3212]
> Before reporting a bug
- [x] Check [common issues](https://katex.org/docs/issues.html).
- [x] Check the bug is reproducible in [the demo](https://katex.org). If not, check KaTeX is up-to-date and installed correctly.
- [x] Search for [existing issues](https://github.com/KaTeX/KaTeX/issues).

**Describe the bug:**
KaTeX seems to fail to find the font for bold/italic greek mathematical symbols

**(La)TeX code:**
* Fails to render:
```latex
𝛼
𝛂
𝜶
```
* Renders correctly:
```latex
α
```

**Expected behavior:**
To render correct versions of character

**Screenshots:**
![imagen](https://user-images.githubusercontent.com/48534497/131267391-cef2c1c6-97a7-4cae-ada4-08f5065f4cb7.png)
![imagen](https://user-images.githubusercontent.com/48534497/131267412-98408859-6d05-4ede-93c0-3bf182097e9c.png)
![imagen](https://user-images.githubusercontent.com/48534497/131267394-d7045ac8-12a9-4872-8d3c-d6c4874b3acc.png)
![imagen](https://user-images.githubusercontent.com/48534497/131267583-31d8f1e4-f368-466e-a2ef-a6846eaa5045.png)

**Environment (please complete the following information):**
 - KaTeX Version: Included with this version of Visual Studio Code and also on the [website](https://katex.org/)
![imagen](https://user-images.githubusercontent.com/48534497/131267466-05dbd3b1-194e-42a5-97ee-53408b57ec4d.png)
 - Device: Desktop
 - OS: Windows 10

**Comment by edemaine** [comment]
@ronkok or @MatiasLGonzalez, do you happen to know how these symbols (Mathematical Italic Small Alpha, Mathematical Bold Small Alpha, Mathematical Bold Italic Small Alpha) are processed in LaTeX (e.g. XeLaTeX)?  It seems natural to implement them as macros that expand to `\mathit{\alpha}`, `\mathbf{\alpha}`, and `\mathit{\boldsymbol{\alpha}}`, respectively, but I'm not sure whether that's the correct font overriding behavior.

**Comment by ronkok** [comment]
> `\mathit{\alpha}`

I don't think it is anywhere nearly that simple. We must keep in mind that Computer Modern does not contain glyphs for such things as mathematical bold small alpha.  A simple LaTeX macro is not enough to get you to where you want.

I believe that XeLaTeX is dependent upon the [unicode-math](https://www.ctan.org/pkg/unicode-math) package. It contains the bridge between TeX and multiple other fonts. It's [processing code](https://ctan.math.illinois.edu/macros/unicodetex/latex/unicode-math/unicode-math-code.pdf) is a great deal more complex than just `\mathit{\alpha}`.

If you are not interested in all the machinery, the `unicode-math` map of Unicode code points to functions can be found at https://github.com/wspr/unicode-math/blob/master/unicode-math-table.tex. I consulted that file a great deal while writing much of KaTeX's Unicode support.

So, to more practical details. Currently, all Unicode code points in the [Unicode Mathematical Alphanumeric Symbols range](https://www.unicode.org/charts/PDF/U1D400.pdf) are handled in the `wide-character.js` file. It successfully handles all the characters in that range that have a glyph in the KaTeX fonts. It does not work well with other characters, such as the ones from this PR.

I might point out that the [KaTeX documentation](https://katex.org/docs/supported.html#letters-and-unicode) explicitly enumerates the Unicode code points that are supported. Lower case bold greek letters are not among them. This PR is properly thought of as a feature request, not a bug report.

To support those missing characters, we either need a font upgrade, or a new approach. I suppose it is possible to work in the manner of `unicodeScripts.js` and create exceptions that call a glyph from Times New Roman and use the metrics of the letter "M".

I have not worked out just how to do that. There is some [branching code](https://github.com/KaTeX/KaTeX/blob/master/src/buildCommon.js#L168-L172) to `wideCharacters`. Perhaps one could write an exception to that exception? It would be nice if life were that easy. It usually isn't.

**Comment by edemaine** [comment]
Sorry, I've been spending too much time with Greek capital letters lately (#3029), which are roman by default but already have glyphs for italic, bold, and bold italic (see #3207 for some screenshots). 
 I forgot that lower-case Greek letters were not so blessed, lacking upright versions; that is issue #564.

However, `\alpha` does have an italic rendering and bold italic rendering:

![image](https://user-images.githubusercontent.com/2218736/131271334-6f9bd559-da22-410d-b427-addf05364a21.png)

So it seems like the italic and bold-italic Greek tables could be supported with existing fonts...

Thanks for the unicode-table link.  It looks like these high characters expand to or are defined as `\mitalpha`, `\mbfalpha`, and `\mbfitalpha` respectively, whereas U+03B1 maps to `\mupalpha` (a symbol we don't have in the fonts).  Do these get mapped into regular LaTeX somehow, or do they just stay in Unicode and get rendered via Unicode fonts?

**Comment by ronkok** [comment]
> However, \alpha does have an italic glyph and bold italic glyph:

Okay, now that I look, I see that such glyphs are there. And there is code in `wideCharacters.js` to call them. So it is not so obvious to me, in that case, where the problem is. But it should be fixable, since the font glyphs exist. I'll look into it.

> Do these get mapped into regular LaTeX somehow, or do they just stay in Unicode and get rendered via Unicode fonts?

I don't know the answer to that. I'm not quite sure what the question is.

**Comment by edemaine** [comment]
Great, thanks!  My question was: does `\mitalpha` have a definition in `unicode-math` other than the one in unicode-math-table.tex, e.g., does it translate to `\mathit{\alpha}` or some other LaTeX expansion?

**Comment by ronkok** [comment]
Dunno.

**Comment by narisuzu** [comment]
![screenshot](https://user-images.githubusercontent.com/16162768/139618074-bb2d093e-30bd-4989-991d-b6c779fa69de.png)
Still have this issue in 0.15.1
